ironfish-cli/src/utils/asset.ts

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { Asset } from '@ironfish/rust-nodejs'
+import { BufferUtils, RpcClient } from '@ironfish/sdk'
+import inquirer from 'inquirer'
+
+export async function selectAsset(
+  client: RpcClient,
+  account: string | undefined,
+  showNativeAsset = true,
+): Promise<string | undefined> {
+  const balancesResponse = await client.getAccountBalances({ account })
+  const assetOptions = []
+
+  let balances = balancesResponse.content.balances
+
+  if (!showNativeAsset) {
+    balances = balances.filter(
+      (balance) => balance.assetId !== Asset.nativeId().toString('hex'),
+    )
+  }
```

Comment:
This part is new, since we disallow burning the native asset, we need to be able to filter it